src/test/scala/org/opennetworkinsight/DNSWordCreationTest.scala

```diff
@@ -0,0 +1,148 @@
+package org.opennetworkinsight
+
+
+import org.opennetworkinsight.testutils.TestingSparkContextFlatSpec
+import org.scalatest.Matchers
+
+class DNSWordCreationTest extends TestingSparkContextFlatSpec with Matchers{
+
+  val countryCodesSet = Set("ac", "ad", "ae", "af", "ag", "ai", "al", "am", "an", "ao", "aq", "ar", "as", "at", "au",
```

Comment:
is this duplicate code with what appears in DNSWordCreation?

we could move it to a separate file and grab it from there